e2e/nomostest/git-server.go

```diff
@@ -138,6 +135,9 @@ func gitDeployment() *appsv1.Deployment {
 		Template: corev1.PodTemplateSpec{
 			ObjectMeta: v1.ObjectMeta{
 				Labels: testGitServerSelector(),
+				Annotations: map[string]string{
```

Comment:
why was this moved?

Reply:
agreed that it's weird that it's showing up on the diff.  This is the incoming change from main.

Reply:
I suggest cleaning up the proposed commit history, this commit comes from another PR

Reply:
I will be squashing on merge.  It's not clear why this is showing up as a diff against main.  https://github.com/GoogleContainerTools/kpt-config-sync/blob/426697e1a2fd0b4298f47fbb4373a0af2b4eb91c/e2e/nomostest/git-server.go#L138

Reply:
By default the auto-submit will perform a squash, but the squashed commit message with all of these commits will be pretty confusing. Would be preferable to squash before the change is submitted so that we can review the commit message

Reply:
let me try to make it all work and make everyone happy and then I will work on the commit history stuff... 😅

Reply:
Some squashed probably more to go